chart/templates/postgrest.yaml

```diff
@@ -40,6 +40,8 @@ spec:
               value: {{ .Values.externalPostgrest.dbAnonRole }}
             - name: PGRST_LOG_LEVEL
               value: {{ .Values.externalPostgrest.logLevel }}
+            - name: PGRST_DB_MAX_ROWS
```

Comment:
I think we might need to pass the env var through to canary-checker for embedded postgrest